modules/googleServices/routes/googleServices.ts

```diff
@@ -0,0 +1,32 @@
+import * as express from 'express';
+import { getGeoreferencia, autocompletar } from '../../../utils/serviciosGoogle';
+
+const router = express.Router();
+
+router.post('/getGooglePoint', async (req, res, next) => {
+    if (req.body.direccion) {
+        try {
+            const resultado: any = await getGeoreferencia(req.body.direccion);
+            res.json(resultado);
+        } catch (err) {
+            return next(err);
+        }
+    } else {
+        return next('Parámetros incorrectos');
+    }
+});
+
+router.get('/getGoogleAutocomplete/:texto', async (req, res, next) => {
```

Comment:
```suggestion
router.get('/autocompletar', async (req, res, next) => {
```

El texto debería ir como query y no como un parametros de la URL.

Recordar cambiar la APP.